js/lib/utils/ItemList.js

```diff
@@ -21,6 +21,21 @@ export default class ItemList {
   }
 
   /**
+   * Check whether the list is empty.
+   *
+   * @returns {boolean}
+   * @public
+   */
+  isEmpty() {
+    for(var prop in this.items) {
+      if(this.items.hasOwnProperty(prop))
```

Comment:
You’re going to want to use `{}` with if blocks to remain consinstant. Also add a space after if.